src/components/font_size_editor/font_size_editor.xml

```diff
@@ -2,14 +2,14 @@
   <t t-name="o-spreadsheet-FontSizeEditor">
     <div class="o-dropdown" t-ref="FontSizeEditor">
       <div
-        class=" o-font-size-editor d-flex align-items-center"
+        class="o-font-size-editor d-flex align-items-center"
         t-att-class="props.class"
-        title="Font Size"
+        t-att-title="title"
         t-on-click="this.toggleFontList">
         <input
           type="number"
-          min="1"
-          max="400"
```

Comment:
why these changes here ?

Reply:
ok, I found `ZoomEditor`.
I'd use composition over inheritance. It's usually easier to read/understand and less coupled together. With inheritance, it can be hard to know what is specific to the child, what is overwritten, what is supposed to overwritten and what not.

Could you instead make a lower level generic component used by both `FontSizeEditor` and `ZoomEditor` ?